_maps/map_files/YogStation/YogStation.dmm

```diff
@@ -45445,6 +45445,23 @@
 	},
 /turf/open/floor/plating,
 /area/engine/atmos)
+"kLr" = (
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/obj/machinery/door/airlock/maintenance{
+	name = "Incinerator Access";
+	req_access_txt = "0";
```

Comment:
Rogue var, we don't want this because it does literally nothing and bloats the map code.
In fastdmm2, just delete the entire var and confirm it to make it go default. (greyed out not white)